components/layout/fragment.rs

```diff
@@ -1655,6 +1655,15 @@ impl Fragment {
                     character_breaking_strategy,
                     max_inline_size,
                     flags)
+            },
+            word_break::T::keep_all => {
```

Comment:
Oops, I didn't mean to post this comment. (I was waffling over whether it was better to keep these separate or not.)  But if you want to combine them, feel free.